vm/systemSmartContracts/staking.go

```diff
@@ -649,8 +649,8 @@ func (r *stakingSC) unBond(args *vmcommon.ContractCallInput) vmcommon.ReturnCode
 		r.eei.AddReturnMessage(fmt.Sprintf("unBond is not possible for key %s which is staked", encodedBlsKey))
 		return vmcommon.UserError
 	}
-	if !r.isActionAllowed(registrationData, args.Arguments[0]) {
-		r.eei.AddReturnMessage("cannot stake node which is jailed or with bad rating " + encodedBlsKey)
+	if r.isNodeJailedOrWithBadRating(registrationData, args.Arguments[0]) {
+		r.eei.AddReturnMessage("cannot unbond node which is jailed or with bad rating " + encodedBlsKey)
```

Comment:
unBond instead to unbond to keep the same pattern returned in messages above

Reply:
done